server/routes/userRoutes.js

```diff
@@ -0,0 +1,79 @@
+const express = require('express');
+const bcrypt = require('bcrypt');
+const database = require('../database');
+
+const router = express.Router();
+
+// Create a new user
+router.post('/', (req, res) => {
+  const user = req.body;
+  user.password = bcrypt.hashSync(user.password, 12);
+  database.addUser(user)
+    .then(user => {
+      if (!user) {
+        return res.send({ error: "error" });
+      }
+
+      req.session.userId = user.id;
+      res.send("🤗");
+    })
+    .catch(e => res.send(e));
+});
+
+// Log a user in
+router.post('/login', (req, res) => {
+  const email = req.body.email;
+  const password = req.body.password;
+
+  database.getUserWithEmail(email)
+    .then(user => {
+      if (!user) {
+        return res.send({ error: "no user with that id" });
+      }
+
+      if (!bcrypt.compareSync(password, user.password)) {
+        return res.send({ error: "error" });
+      }
+
+      req.session.userId = user.id;
+      res.send({
+        user: {
+          name: user.name,
+          email: user.email,
+          id: user.id
+        } 
+      });
+    });
+});
+
+// Log a user out
+router.post('/logout', (req, res) => {
+  req.session.userId = null;
+  res.send({});
+});
+
+// Return information about the current user (based on cookie value)
+router.get("/me", (req, res) => {
```

Comment:
It's not a huge deal, but it might help to run this through `prettier` to get the single vs. double quotes consistent across this MR.